clapper/clapper.ml

```diff
@@ -0,0 +1,7 @@
+module Clapper = struct
+
+  let punctuate (lst : string list) (sep : string) : string =
```

Comment:
`Sys.args` comes in as a `string array`. When should I use `array` and when should I use `list`?